src/utilities/host-utils.ts

```diff
@@ -0,0 +1,12 @@
+export const isOfflineHost = (host: string) => (host === "localhost:11002") || (host === "activity-player-offline.concord.org");
+
+export const getCanonicalHostname = () => {
+  if(window.location.hostname === "activity-player-offline.concord.org") {
+    return "activity-player.concord.org";
+  }
+  return window.location.hostname;
+};
+
+export const isProductionOrigin = (origin: string) => {
+  return origin === "https://activity-player.concord.org" || origin === "https://activity-player-offline.concord.org";
```

Comment:
```suggestion
  return getCanonicalHostname(origin) === "https://activity-player.concord.org";
```
DRY

Reply:
This won't work because in one case it is hostname in the other case it the origin (which includes the protocol).
We could strip the protocol and then use getCanonicalHostname, but I'm reluctant to add any string based url manipulation. So this would mean making a URL object and getting the hostname. At which point just repeating our selves seems just as good.